code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -389,10 +389,9 @@
 			src.changeNext_move(CLICK_CD_MELEE)
 			if(heal_cooldown <= world.time && !stat)
 				var/mob/living/carbon/C = target
-				C.adjustBruteLoss(-5)
-				C.adjustFireLoss(-5)
-				C.adjustOxyLoss(-5)
-				C.adjustToxLoss(-5)
+				C.heal_overall_damage(amount, amount, 10, 10)
```

Comment:
This is bad - you want `C.heal_overall_damage(10, 10, 0, 1)` - `amount` was a placeholder there.
The third argument is whether this can heal internal bleeding or not - this probably SHOULDN'T, so that third argument needs to be 0.